docs/hashes.toml

```diff
@@ -1,42 +1,72 @@
 # Generated by: ckb cli hashes -b
 [ckb_dev]
-genesis = "0x20547a218a969cddda11d66a4a4c2ff61fdf6901652364d7cf461567d23cf160"
-cellbase = "0xf543d348bcfba344d50e5923ec4965099e811ca5da80c32975e26de1c3c7e62d"
+genesis = "0x439fb2d2b6a1205ed9a8fcf1580bae935b016c9263a47fcadb29eacd3668bd2a"
+cellbase = "0xceca4433fa418e924f57aedf6cd8213a9ed36b864aefe399a4d1147ca5539782"
 
 [[ckb_dev.system_cells]]
-path = "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
+path = "Bundled(specs/cells/secp256k1_data)"
 index = 1
+code_hash = "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4"
+
+[[ckb_dev.system_cells]]
+path = "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
+index = 2
 code_hash = "0x664b03c55410dd7d694a73779fa86edd7f069ee17fd79b0fef4c4fec0377b9a6"
 
+[[ckb_dev.system_cells]]
+path = "Bundled(specs/cells/secp256k1_ripemd160_sha256_sighash_all)"
+index = 3
+code_hash = "0x1acf88382192fbf76fc5a457a11cc393a90d84a46c5e0d7ceebfd1d0dec871e4"
+
 [[ckb_dev.system_cells]]
 path = "Bundled(specs/cells/dao)"
-index = 2
+index = 4
 code_hash = "0x75ff25b427a9bee13106da176f1b5884f931432a9bf0e06fa4044fe6bc327cd5"
 
 [ckb_staging]
-genesis = "0xebdca80c83260cee3013a210372c4d934631f948bbd044eafdb448d6564cc487"
-cellbase = "0x3c1efe9a3c9ca1a008a5a52be9084a38faa88e8b8fedbc247284c847e519d66d"
+genesis = "0x6853d356bec5eede3c7e6dccba18c1008570930e7ecca8282d6074e856777d3e"
+cellbase = "0x3ff78c6d260ca3a900d6bb9da81784cfe9cd9e3d480a135483b64067fb4a4616"
 
 [[ckb_staging.system_cells]]
-path = "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
+path = "Bundled(specs/cells/secp256k1_data)"
```

Comment:
Please do not change the position of the existing cells.

1. secp256k1_blake160_sighash_all
2. dao
3. secp256k1_data
4. secp256k1_ripemd160_sha256_sighash_all

Reply:
TODO: we should include the second transaction hash in hashes.toml

Reply:
ok